rust/lance-index/src/vector/hnsw/builder.rs

```diff
@@ -226,7 +226,7 @@ impl HNSW {
         match self.inner.visited_generator_queue.push(visited_generator) {
             Ok(_) => {}
             Err(_) => {
-                println!("visited_generator_queue is full");
+                panic!("visited_generator_queue is full");
```

Comment:
@BubbleCal @broccoliSpicy Seems like we originally didn't want to panic here. Why is it acceptable to panic here? Should we instead make this a warning?

```suggestion
                log::warn!("visited_generator_queue is full");
```

Reply:
yeah, sorry, this is due to a miscommunication between me and @BubbleCal.